frontend/server/src/main/java/org/pytorch/serve/http/HttpRequestHandler.java

```diff
@@ -61,6 +61,9 @@ protected void channelRead0(ChannelHandlerContext ctx, FullHttpRequest req) {
         } catch (OutOfMemoryError e) {
             logger.trace("", e);
             NettyUtils.sendError(ctx, HttpResponseStatus.INSUFFICIENT_STORAGE, e);
+        } catch (IllegalArgumentException e) {
+            logger.error("", e);
+            NettyUtils.sendError(ctx, HttpResponseStatus.FORBIDDEN, e);
```

Comment:
As per #196, it should be 404 and not 403. Why 403? Also, let's keep things simple i.e. Catch ModelVersionNotFound exception in NOT-FOUND if clause above instead of having base as illegalArgexception.

Reply:
ModelVersionNotFound extends ModelNotFound exception, which is already handled in the first catch block above. This is just an extra exception handling.